spark-common/src/main/java/com/k15t/spark/atlassian/AtlassianAppServlet.java

```diff
@@ -72,40 +69,13 @@ protected String renderVelocity(String template, RequestProperties props) throws
 
     @Override
     protected String prepareIndexHtml(String indexHtml, RequestProperties props) throws IOException {
-
-        Document document = Jsoup.parse(indexHtml, props.getUri().toString());
-
         if (!isDevMode()) {
```

Comment:
Minor / opinion: maybe it's just my eyes ignoring the "!" here but I first thought that the if block is only run in "dev mode" and not the other way around

Maybe this could work too
`if (isDevMode()) {
   return indexHtml;
} else {
   // apply cache keys and return the result
}
`